.github/CODEOWNERS

```diff
@@ -100,3 +100,4 @@
 /retail                          @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/cloud-retail-team
 /unittests                       @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/serverless-runtimes
 /bigquery/bigquerydatatransfer   @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/bigquery-data-connectors
+/routeoptimization               @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/geo-routeoptimization
```

Comment:
I've added the geo-routeoptimization team, to make this a valid CODEOWNERS file, looks like @GoogleCloudPlatform/bigquery-data-connectors needs to get removed from the above line.